deps/graph-parser/src/logseq/graph_parser/exporter.cljs

```diff
@@ -1008,8 +1019,9 @@
                                       (not (:block/file %))))
                         ;; remove file path relative
                         (map #(dissoc % :block/file)))
-        ;; Fetch all named ents once per import file to speed up named lookups
-        all-existing-page-uuids (get-all-existing-page-uuids @conn @(:classes-from-property-parents import-state))
+        ;; Build all named ents once per import file to speed up named lookups
+        all-existing-page-uuids (get-all-existing-page-uuids @(:classes-from-property-parents import-state)
```

Comment:
I diffed this uuid with the previous db approach and found minor differences I ran out of time to investigate. I imported a couple of local graphs and couldn't found any behaviorial bugs caused by this difference